src/components/ui/ProgressPanel.svelte

```diff
@@ -54,19 +54,68 @@
       default: 'View report'
     }),
     CONFORMANCE_LEVEL: $translate('WCAG.COMMON.CONFORMANCE_LEVEL'),
-    REPORT_FOR: $translate('UI.REPORT.REPORT_FOR')
+    REPORT_FOR: $translate('UI.REPORT.REPORT_FOR'),
+    REPORTED_ON: $translate('UI.REPORT.REPORTED_ON'),
+    REPORTED_ON_OF: $translate('UI.REPORT.REPORTED_ON_OF')
   };
 
-  $:  console.log('totalevaluated', totalEvaluated);
   $: conformanceTarget = $scopeStore['CONFORMANCE_TARGET'];
-  $: wgacVersion = $scopeStore['WCAG_VERSION'];
-  $: percentageEvaluated = (totalEvaluated / totalToEvaluate) * 100;
+  $: wcagVersion = $scopeStore['WCAG_VERSION'];
+  $: percentageTotalEvaluated = (totalEvaluated / totalToEvaluate) * 100;
 
   $: principles = [...new Set($wcag.map((a) => a.num.split('.')[0]))];
 
-  $: console.log($assertions);
+  $: upToWcagVersion = wcagVersion === "2.0" ?
+    ["2.0"] : ["2.0", "2.1"];
 
-  let totalEvaluated = 3;
+  $: filteredCriteria = {
+   1: $wcag.filter(item => item.num.startsWith("1.")).filter(isInScope) || {},
```

Comment:
Filtering on version is already done; `$wcag` returns criteria based on `scope.wcag_version`

Reply:
ah, so I could remove `.filter(isInScope)` ?

Reply:
I tried this but it returns totals that are too large, leaving `isInScope` filter in for now to be sure the numbers are correct